full-backend-tests/src/test/java/org/graylog/plugins/views/SuggestionResourceIT.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.plugins.views;
+
+import io.restassured.response.ValidatableResponse;
+import io.restassured.specification.RequestSpecification;
+import org.graylog.testing.completebackend.GraylogBackend;
+import org.graylog.testing.containermatrix.ContainerVersions;
+import org.graylog.testing.containermatrix.SearchServer;
+import org.graylog.testing.containermatrix.annotations.ContainerMatrixTest;
+import org.graylog.testing.containermatrix.annotations.ContainerMatrixTestsConfiguration;
+import org.graylog.testing.utils.GelfInputUtils;
+import org.graylog.testing.utils.SearchUtils;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.graylog.testing.completebackend.Lifecycle.CLASS;
+import static org.graylog.testing.containermatrix.SearchServer.ES6;
+import static org.graylog.testing.containermatrix.SearchServer.ES7;
+import static org.graylog.testing.containermatrix.SearchServer.OS1;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+@ContainerMatrixTestsConfiguration(serverLifecycle = CLASS, extraPorts = SuggestionResourceIT.GELF_HTTP_PORT, searchVersions = {ES6, ES7, OS1})
```

Comment:
change to Lifecyle.VM? also remove searchVersions (all ist default), GELF port is added as a default to, but for transparency, it could be left in

Reply:
Done, good catch, thx!